lib/chat/actions.tsx

```diff
@@ -127,22 +127,22 @@ async function submitUserMessage(content: string) {
   let textNode: undefined | React.ReactNode
 
   const result = await streamUI({
-    model: openai('gpt-3.5-turbo'),
+    model: openai('gpt-4o-mini'),
```

Comment:
Like @MildTomato said with the GitHub auth, I don't know if we should touch features not related to Supabase here.

Reply:
Agreed. Don't change.